src/scripts/main.js

```diff
@@ -2,7 +2,23 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const tableElement = document.querySelector('.dashboard');
 
-// write your code here
+people.forEach(person => {
+  const row = tableElement.insertRow();
+
+  const cells = [];
+
+  [0, 1, 2, 3, 4, 5].forEach((cell) => {
+    cells.push(row.insertCell(cell));
+  });
```

Comment:
it's better to move 0, 1, 2, 3, 4 and to the constants, as it will make a code cleaner
for instance:
`const NAME_COLUMN = 1;`